README.md

```diff
@@ -60,17 +68,16 @@ Any advise for common problems or issues.
 [@Jordy Ampe](https://github.com/JordyGit)  
 [@RobbertVH](https://github.com/RobbertVH)
 
-Project Link: [chat-integration](https://github.com/SkylineCommunications/chat-integration)
+Project Link: [Chat-integration](https://github.com/SkylineCommunications/chat-integration)
 
 ## Version History
 
-* 0.2
-   * optimizations
-   * See [commit change]() or See [release history]()
-* 1.0
-   * Initial Release
+- 0.2
+  - Refactoring
+  - Generalizations
+- 1.0
+  - Initial Release
```

Comment:
We shouldn't have these things here? we haven't released it yet so should be empty still
we don't want commit messages here?